.github/workflows/test.yml

```diff
@@ -50,7 +50,7 @@ jobs:
         run: python -m pip install --no-deps -e . # `-e` required for correct `coverage` run.
 
       - name: Run tests
-        run: pytest
+        run: pytest -m 'not ipu'
```

Comment:
Do you think that flag should be added in `pyproject.toml` so people running the tests locally (without IPU) will work (and also match the CI behavior)?

Reply:
This is a good idea - I'll check how it works with overwriting